sample/src/main/java/io/getstream/chat/example/LoginActivity.java

```diff
@@ -0,0 +1,52 @@
+package io.getstream.chat.example;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ListView;
+import android.widget.Toast;
+
+
+import io.getstream.chat.example.adapter.UserListItemAdapter;
+import io.getstream.chat.example.utils.UserStorage;
+
+public class LoginActivity extends AppCompatActivity {
+
+    private static final String TAG = LoginActivity.class.getSimpleName();
+
+    private UserListItemAdapter adapter;
+    private ListView lv_users;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+
+        lv_users = findViewById(R.id.lv_users);
+        UserStorage.init(this);
```

Comment:
Lets move it to application. `LoginActivity` shouldn't care about initialisation of other parts of the app.